src/sentry/tasks/reprocessing2.py

```diff
@@ -64,6 +64,7 @@ def reprocess_group(
         batch_size=settings.SENTRY_REPROCESSING_PAGE_SIZE,
         state=query_state,
         referrer="reprocessing2.reprocess_group",
+        tenant_ids={"organization_id": models.Project.objects.get(id=project_id).organization_id},
```

Comment:
this one could be problematic as this is an ingest flow. would check with @untitaker on if this is okay to do here.

Reply:
reprocessing is not perf sensitive, but if you can, use `get_from_cache` across the board just because it's low-effort